SConstruct

```diff
@@ -12,13 +12,19 @@ else:
 
 sources = Glob("src/*.cpp")
 
+steam_audio_lib_path = env.get("STEAM_AUDIO_LIB_PATH", "")
```

Comment:
I don't know much about scons, but assuming the second arg in get() is the default value, could we set it to `"src/lib/steamaudio/lib"` and remove the if below? Thanks

Reply:
Good point. 
Fixed